src/main/java/com/cleanroommc/groovyscript/compat/mods/tinkersconstruct/Melting.java

```diff
@@ -20,12 +24,31 @@
 import slimeknights.tconstruct.library.smeltery.MeltingRecipe;
 
 import java.util.List;
+import java.util.Map;
 import java.util.stream.Collectors;
 
-public class Melting extends MeltingRecipeRegistry {
+@RegistryDescription
+public class Melting extends MeltingRecipeRegistry implements IDynamicGroovyProperty {
```

Comment:
could you split Melting into `Melting` and `EntityMelting` classes? it doesnt seems overcomplicated for this to work via this nesting behavior.